sksports/model/tests/test_heart_rate.py

```diff
@@ -0,0 +1,53 @@
+"""Tests the module used to estimate the heart-rate."""
+
+# Authors: Aart Goossens
+#          Guillaume Lemaitre
+# License: MIT
+
+import pytest
+import numpy as np
+import pandas as pd
+
+from sksports.model import exp_heart_rate_model
+from sksports.model import HeartRateRegressor
+
+POWER = np.array([100] * 10 + [240] * 100 + [160] * 100)
```

Comment:
You can consider moving these constants to a pytest fixture. It's a matter of preference but I think these fixtures are really nice to work with and also well reusable in other tests. But I'm fine with doing improvements like this later.